README.md

```diff
@@ -44,10 +44,10 @@ The following environment variables can be used optionally to configure your Tes
 | Environment Variable | Default Value | Description |
 | ---- | ---- | ----------- |
 | `BUILDKITE_SPLITTER_DEBUG_ENABLED` | `false` | Flag to enable more verbose logging. |
-| `BUILDKITE_SPLITTER_RETRY_CMD` | `BUILDKITE_SPLITTER_TEST_CMD` with `--only-failures` | The command to rerun the failed tests. For Rspec, it depends on `example_status_persistence_file_path` to be defined as part of your Rspec configuration for the retry to work. Please see [Rspec documentation](https://rspec.info/features/3-12/rspec-core/command-line/only-failures/) for available retry options.
-| `BUILDKITE_SPLITTER_RETRY_COUNT` | `0` | The number of retries performed. Test splitter runs the test command defined in `BUILDKITE_SPLITTER_TEST_CMD`, and retries only the failed tests for a maximum of `BUILDKITE_SPLITTER_RETRY_COUNT` times with the retry command `BUILDKITE_SPLITTER_RETRY_CMD`. |
+| `BUILDKITE_SPLITTER_RETRY_CMD` | `BUILDKITE_SPLITTER_TEST_CMD` | The command to retry the failed tests. Test Splitter will fill in the `{{testExamples}}` placeholder with the failed tests. If not set, the Test Splitter will use the same command defined in `BUILDKITE_SPLITTER_TEST_CMD`. |
+| `BUILDKITE_SPLITTER_RETRY_COUNT` | `0` | The number of retries. Test Splitter runs the test command defined in `BUILDKITE_SPLITTER_TEST_CMD` and retries only the failed tests up to `BUILDKITE_SPLITTER_RETRY_COUNT` times, using the retry command defined in `BUILDKITE_SPLITTER_RETRY_CMD`. |
 | `BUILDKITE_SPLITTER_SPLIT_BY_EXAMPLE` | `false` | Flag to enable split by example. When this option is `true`, the Test Splitter will split the execution of slow test files over multiple partitions. |
-| `BUILDKITE_SPLITTER_TEST_CMD` | `bundle exec rspec {{testExamples}}` | Test command to run your tests. Test splitter will fill in the `{{testExamples}}` placeholder with the test splitting results |
+| `BUILDKITE_SPLITTER_TEST_CMD` | `bundle exec rspec --format progress {{testExamples}}` | Test command to run your tests. Test Splitter will fill in the `{{testExamples}}` placeholder with the test splitting results. |
```

Comment:
I'm curious why we added `--format progress` to the default test command. Do we need to add `--format progress` to the test command if user is not using the default one?

Reply:
Since we are adding `--format json --out xxx.json` to save the json file, no output will be printed if the users use the default command because the `json` format we are appending overrides Rspec default format which is `progress`. Therefore, I added `--format progress` to the default command, so by default it prints the progress to stdout and save the json to a file.

Reply:
Oh yeah I think I kinda followed.
So say if a user overwrites `BUILDKITE_SPLITTER_TEST_CMD` with something like `bin/rspec` do we need to append a `--format progress` to the `bin/rspec`? Or you already doing it 😅

Reply:
If the customer change the default to `bin/rspec` it won't have `--format progress`. I'll update the documentation to tell customer to set their own formatter when using the custom command.